phpcs.xml

```diff
@@ -14,4 +16,24 @@
 	<rule ref="WordPress">
 		<exclude name="WordPress.VIP" />
 	</rule>
+
+	<rule ref="WordPress.WP.I18n">
+		<properties>
+			<!-- TGMPA itself uses `tgmpa`, the example file uses `theme-slug`. -->
+			<property name="text_domain" type="array" value="tgmpa,theme-slug" />
+		</properties>
+	</rule>
+
+	<rule ref="WordPress.WP.DeprecatedFunctions">
+		<properties>
+			<property name="minimum_supported_version" value="3.7" />
+		</properties>
+	</rule>
+
+	<rule ref="WordPress.WP.DeprecatedFunctions">
```

Comment:
Sorry, goes to show that I'm working on too many PRs at the same time. The second one should be for `WordPress.WP.DeprecatedParameters`. Adjusting now.